src/gui/addnewtorrentdialog.ui

```diff
@@ -189,6 +189,16 @@
     </widget>
    </item>
    <item>
+    <widget class="QCheckBox" name="doNotDeleteTorrentCheckBox">
+     <property name="toolTip">
+      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;When checked, the .torrent file will not be deleted despite the settings in the quot;Downloadquot; page of the options dialog.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
```

Comment:
why html tags here? Is pure text not enough?
anyway, for me, `quot;` isn't replaced to `"`